Wikipedia/Localizations/hu.lproj/Localizable.strings

```diff
@@ -37,6 +37,7 @@
 "zero-free-verbiage" = "Ingyenes Wikipédia-hozzáférés a mobilszolgáltatódtól (adatforgalmi díjak elengedve)";
 "zero-charged-verbiage" = "A Wikipédia Zero ki van kapcsolva";
 "zero-charged-verbiage-extended" = "Más szócikkek betöltése adatforgalmi díjakkal járhat. Az elmentett szócikkek kapcsolat nélkül is elérhetőek, így ingyen megtekinthetőek.";
+"zero-webpage-url" = "https://wikimediafoundation.org/wiki/Wikipedia_Zero_App_FAQ";
```

Comment:
Consider moving this string out of translations as it's identical across languages. Consider changing the URL to mobile, https://m.wikimediafoundation.org/wiki/Wikipedia_Zero_App_FAQ.